### PR TITLE
Frontpage: update blocks

### PR DESCRIPTION
**Pull request by Pineirin**
closes: https://github.com/inveniosoftware/demo-inveniordm/issues/104

### CODE REVIEW

File: demo-inveniordm/celerybeat-schedule.bak [hunk 1]
```diff
@@ -0,0 +1,4 @@
+'entries', (2048, 1023)
```
Comment (kpsherva) [inline demo-inveniordm/celerybeat-schedule.bak:1]: this should not be commited

File: demo-inveniordm/templates/demo_inveniordm/demo_frontpage.html [hunk 2]
```diff
@@ -10,14 +10,13 @@
...
-{%- block first_section %}
-
+{%- block intro_section %}
   <section class="invenio-rdm-project-section section-content white-bg">
     <h2 class="section-title">Sandbox deployment of InvenioRDM</h2>
     <div class="ui container two column grid center aligned vertically padded relaxed stackable">
       <div class="row one column">
         <div class="column">
-          <p>Here you can test all the features it has to offer!</p>
+          <p>Here you can test all the features it has to offerrrrrrrrrr!</p>
```
Comment (zzacharo) [inline demo-inveniordm/templates/demo_inveniordm/demo_frontpage.html:19]: Leftover

**Review by zzacharo — COMMENTED** [review]
All the celerybeat-schedule tasks should be removed or added in .gitignore!

**Comment by zzacharo** [comment]
@Pineirin can you also please add screenshots with before/after the change of blocks so we can test the visual part quickly?

File: demo-inveniordm/docker-compose.full.yml [hunk 3]
```diff
@@ -75,15 +75,6 @@ services:
       - static_data:/opt/invenio/var/instance/static
       - uploaded_data:/opt/invenio/var/instance/data
       - archived_data:/opt/invenio/var/instance/archive
-    depends_on:
-      es:
```
Comment (kpsherva) [inline demo-inveniordm/docker-compose.full.yml:79]: shouldn't we change `es:` to ` search:` and not to remove dependencies? ping @jrcastro2

File: demo-inveniordm/docker-compose.full.yml [hunk 4]
```diff
@@ -97,25 +88,18 @@ services:
...
-        condition: service_started
-      mq:
-        condition: service_started
+
   # Worker
   worker:
     extends:
       file: docker-services.yml
       service: app
-    restart: "always"
```
Comment (kpsherva) [inline demo-inveniordm/docker-compose.full.yml:114]: why restart is also removed?

File: demo-inveniordm/docker-services.yml [hunk 5]
```diff
@@ -29,14 +29,6 @@ services:
       - "INVENIO_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://demo-inveniordm:demo-inveniordm@db/demo-inveniordm"
       - "INVENIO_WSGI_PROXIES=2"
       - "INVENIO_RATELIMIT_STORAGE_URL=redis://cache:6379/3"
-  lb:
```
Comment (kpsherva) [inline demo-inveniordm/docker-services.yml:32]: why is lb removed?

File: demo-inveniordm/docker-services.yml [hunk 6]
```diff
@@ -80,42 +72,30 @@ services:
...
     restart: "unless-stopped"
     environment:
+      # settings only for development. DO NOT use in production!
       - bootstrap.memory_lock=true
       - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
-      - discovery.type=single-node
-      - plugins.security.disabled=true # NOTE: this shouldn't be used in production!
-      - "DISABLE_SECURITY_PLUGIN=true"
       - "DISABLE_INSTALL_DEMO_CONFIG=true"
-    healthcheck:
```
Comment (kpsherva) [inline demo-inveniordm/docker-services.yml:89]: how do we check if redis is ready if we remove the healthcheck

File: demo-inveniordm/docker-services.yml [hunk 7]
```diff
@@ -80,42 +72,30 @@ services:
...
     ports:
       - "5601:5601"
     expose:
       - "5601"
     environment:
-      - "OPENSEARCH_HOSTS=http://search:9200"
+      # settings only for development. DO NOT use in production!
+      - 'OPENSEARCH_HOSTS=["http://search:9200"]'
       - "DISABLE_SECURITY_DASHBOARDS_PLUGIN=true"
-  flower:
```
Comment (kpsherva) [inline demo-inveniordm/docker-services.yml:117]: why do we remove flower?

**Review by kpsherva — APPROVED** [review]

**Pull request merged**